### PR TITLE
Update references to old-style arguments

### PR DESCRIPTION
**Pull request by anowlcalledjosh**
Updating a few more old spellings of arguments.

I also noticed some references to `-scatterTranches` and `-sampleEvery`, but as far as I can tell they're not currently valid arguments to anything, so I didn't touch the references to those.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java [hunk 1]
```diff
@@ -268,7 +268,7 @@ private void validateAndInitializeArgs() {
         Utils.validateArg(hcArgs.likelihoodArgs.BASE_QUALITY_SCORE_THRESHOLD >= QualityUtils.MIN_USABLE_Q_SCORE, "BASE_QUALITY_SCORE_THRESHOLD must be greater than or equal to " + QualityUtils.MIN_USABLE_Q_SCORE + " (QualityUtils.MIN_USABLE_Q_SCORE)");
 
         if ( emitReferenceConfidence() && samplesList.numberOfSamples() != 1 ) {
-            throw new CommandLineException.BadArgumentValue("--emitRefConfidence", "Can only be used in single sample mode currently. Use the sample_name argument to run on a single sample out of a multi-sample BAM file.");
+            throw new CommandLineException.BadArgumentValue("--emit-ref-confidence", "Can only be used in single sample mode currently. Use the sample-name argument to run on a single sample out of a multi-sample BAM file.");
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java]: maybe this should be `--sample-name`

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNVariantWriteTensors.java [hunk 2]
```diff
@@ -47,22 +47,22 @@
  * gatk CNNVariantWriteTensors \
  *   -R reference.fasta \
  *   -V input.vcf.gz \
- *   -truth-vcf platinum-genomes.vcf \
- *   -truth-bed platinum-confident-region.bed \
- *   -tensor-name reference \
- *   -output-tensor-dir my-tensor-folder
+ *   --truth-vcf platinum-genomes.vcf \
+ *   --truth-bed platinum-confident-region.bed \
+ *   --tensor-type reference \
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNVariantWriteTensors.java]: There is another reference to `tensor-name` in the documentation above here and in CNNVariantTrain which would be good to update as well.

**Comment by anowlcalledjosh** [comment]
@lbergelson No problem! I think I addressed the two issues you mentioned.

**Comment by lbergelson** [comment]
@anowlcalledjosh Thank you.

**Pull request merged**